src/open_inwoner/cms/plugins/tests/test_appointments.py

```diff
@@ -19,12 +20,14 @@ def test_plugin(self, m):
 
         self.assertTrue(data.user.has_verified_email())
 
-        html, context = cms_tools.render_plugin(
-            UserAppointmentsPlugin, plugin_data={}, user=data.user
-        )
+        with freeze_time("2020-01-01 00:00"):
+            html, context = cms_tools.render_plugin(
+                UserAppointmentsPlugin, plugin_data={}, user=data.user
+            )
 
         appointments = context["appointments"]
 
+        # past appointments should not show up
         self.assertEqual(len(appointments), 2)
```

Comment:
This might be more explicit (the `notIn` follows from the previous one, but code > comments). But it's a nitpick.

```suggestion
        appointment_titles = [a.title for a in appointments]
        self.assertEqual([data.appointment_idcard.title, data.appointment_passport.title], appointment_titles)
        self.assertNotIn(appointment_titles, data, appointment_old.title)
```

Reply:
You're right. I changed it, though using a slightly different approach.